graphql_compiler/schema_generation/utils.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+def amend_custom_scalar_types(schema, scalars):
+    """Amend the serialization, parsing and description of custom scalar types in the schema.
+
+       Information about the fields, (other than the name), of custom GraphQLScalarType objects is
+       lost when GraphQLSchema objects are serialized into strings. This function uses the
+       original type definitions to mutate the schema and reconstructs the fields of custom
+       scalar types.
+
+       Args:
+           schema: GraphQLSchema object
```

Comment:
Missing a big, important fact here: this input argument is mutated! I know the text above mentions it, but it's important to mention that in every place that a reader might reference.